src/main/java/seedu/task/commons/util/StringUtil.java

```diff
@@ -31,7 +31,9 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
         for (String wordInSentence: wordsInPreppedSentence) {
-            if (wordInSentence.equalsIgnoreCase(preppedWord)) return true;
+            //@@joshuaching A0163673Y
+            if (wordInSentence.toLowerCase().contains(preppedWord.toLowerCase())) return true;
```

Comment:
This means that search isn't case sensitive, right? The user guide would need to be updated. : )